A14-channelz.md

```diff
@@ -285,10 +282,10 @@ Calls may send and receive any number of messages, including zero.
 
 The number of keepalives sent represents how often the local endpoint wants to
 keep the socket alive.  Generally this is an HTTP/2 PING frame.  However, not
-all PING frames are keep alives.  Specifically, pings that were sent as a reply
-(ACK bit set) or were used for reasons unrelated to keep alives should not count
+all PING frames are kept alive.  Specifically, pings that were sent as a reply
```

Comment:
The original way is correct.  A keep alive is a noun in this context.

Reply:
Oh Got it.